src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+**ARO**
+
+* Add Public and Private to params for help with ingress/apiserver visibility (#13550)
+
```

Comment:
You no longer need to modify HISTORY.rst manually. It will be generated from PR title or description.

Reply:
I can revert this commit to allow this to be generated.